docs/_includes/partials/component/footer.njk

```diff
@@ -1,3 +1,8 @@
+<script data-helmet type="module">
+  import '@rhds/elements/rh-footer/rh-footer-universal.js';
+  import '@rhds/elements/rh-footer/rh-footer-copyright.js';
+</script>
```

Comment:
These kinds of changes that move these deps to the file they are needed in are 👨‍🍳 💋